source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
+Linux OS for Intel Architecture with the following bundle installed. If you
+don't have it already, you can add it with the :command:`swupd bundle-add`
+command like so::
```

Comment:
Please use the code-block directive and specify the language as bash. A good guide to differentiate between when to use bash and when to use console as a language is: If the text is the console output, use console, otherwise use bash.